cpp/src/centrality/betweenness_centrality_impl.cuh

```diff
@@ -221,8 +233,10 @@ void accumulate_vertex_results(
   bool with_endpoints,
   bool do_expensive_check)
 {
+  
   constexpr vertex_t invalid_distance = std::numeric_limits<vertex_t>::max();
 
+  RAFT_CUDA_TRY(cudaDeviceSynchronize());
```

Comment:
I guess you added this for performance measurement, don't forget to delete this before merging this PR.